### ISSUE TITLE
System Tray Icon Does Nothing

### ISSUE THREAD

**Issue by aeridus** [keybase/keybase-issues#2813]
I'm on Windows 7 64-bit, and currently the system tray icon does nothing on left click or right click. I expect it to bring up the full app on left click and a context menu on right click. I'm not using Aero, if that makes a difference.

**Comment by zanderz** [comment]
Does it not show the "widget"? 
![image](https://cloud.githubusercontent.com/assets/862397/22764628/a7285dcc-ee20-11e6-976f-9bcc4b3d7676.png)

**Comment by aeridus** [comment]
Nope. I uninstalled and reinstalled but that didn't seem to help. I get a tab on my taskbar with Keybase and the black Keybase icon, but when I try clicking on it it disappears. I'm wondering if it's trying to show a popup that requires Aero.

**Comment by robyf** [comment]
I have the exact same issue with the same setup: Windows7 64-bit, no aero.

**Comment by MichaelQuell** [comment]
Same issue for me

**Comment by zanderz** [comment]
We'll work more on a repro. I'm not sure the logs will show anything but can someone do `keybase log send` just in case? Thanks.

**Comment by aeridus** [comment]
Log sent! My log ID is: 11096123cf7fb5b45a24f61c

I made sure to create the log shortly after attempting to use the system tray icon.

**Comment by robyf** [comment]
Sent, log ID: ec9b2b8dbcb538ec4ee22a1c

What I did was:
- Turn on the machine
- Close the keybase window
- Click on the icon in the system tray
- Click on the keybase icon in the task bar (that disappeared immediately)

**Comment by rpottsoh** [comment]
Using windows 7 32 bit.  Tray icon also does nothing.  Tray icon's hair does turn red when a file arrives in shared folders.  Left/Right clicking causes keybase to show up appear on taskbar but no window appears.  Upon clicking ANYWHERE else taskbar appearance goes away.  Not sure what version of keybase is installed, I do not see an `About` anywhere that would indicate.  
Version information from keybase shell:
`Client:  1.0.18-20170208085140+dc66461`
`Service: 1.0.18-20170208085140+dc66461`

**Comment by rpottsoh** [comment]
Keybase just updated, now running
`Client:  1.0.18-20170216112201+a5b6199`
`Service: 1.0.18-20170216112201+a5b6199`

No change to tray icon behavior.

**Comment by novas0x2a** [comment]
Same problem, no aero, {left,right}-clicking tray icon does nothing, and I can't unminimize the client window (seems permanently hidden?). Log is 478abf87eb0932b0bac6161c, shell says:
```
Client:  1.0.24-20170619073621+91a6943
Service: 1.0.24-20170619073621+91a6943
```

**Comment by novas0x2a** [comment]
If I turn aero on, everything seems to work as expected, so yeah, pretty sure the problem is that the keybase gui is aero-only in windows.

**Comment by robyf** [comment]
I'm afraid it's related to this: https://github.com/electron/electron/issues/1821

The electron developers (keybase uses electron for the GUI) are not willing to even consider looking into this matter :(

**Comment by ericfrederich** [comment]
I'm experiencing this as well.  On my work machine the GUI works, but on my personal one I have this issue.  Both are Windows 10.

I see an icon, right clicking or left clicking does nothing except bring up a taskbar icon.  Clicking this icon makes it disappear, so does alt-tab'ing to it.

I changed my zoom level to the recommended 200% from the 125% I had it at; nothing.
Uninstall/reinstall; nothing
